cabal.project

```diff
@@ -27,3 +27,22 @@ constraints:
       base-unicode-symbols < 0.2.4,
       megaparsec < 8,
       neat-interpolation < 0.4
+
+-- GHC-8.8
+
+package pact
+    ghc-options: -Wwarn
+    haddock-options: --optghc=-Wwarn
+
+allow-newer:
+    *:sbv
```

Comment:
this clause may be unnecessary once https://github.com/kadena-io/pact/pull/783 goes in.

Reply:
👍 nice